src/js/components/Meter/Bar.js

```diff
@@ -49,30 +67,49 @@ const Bar = forwardRef((props, ref) => {
             someHighlight && !highlight ? background : colorName,
             theme,
           )}
-          strokeWidth={height}
+          strokeWidth={direction === 'horizontal' ? thickness : length}
           strokeLinecap={round ? 'round' : 'butt'}
           {...hoverProps}
           {...pathRest}
         />
       );
-    })
-    .reverse(); // reverse so the caps looks right
```

Comment:
I think we still need this `reverse()` to ensure rounded caps look right. I think this is the source of the unexpected snapshot changes.